sqlglot/parser.py

```diff
@@ -923,6 +923,101 @@ class Parser(metaclass=_Parser):
         TokenType.START_WITH: lambda self: ("connect", self._parse_connect()),
     }
 
+    OPTION_PREFIX_TO_PARSE_FUNCTION = {
```

Comment:
Let's not pollute the base parser with this - it's very specific to T-SQL, so imo we should move it there.

Reply:
Done